packages/strapi-plugin-open-forms-embed/admin/src/components/CustomCombobox/index.tsx

```diff
@@ -39,7 +39,9 @@ function CustomCombobox({
   const [isLoading, setIsLoading] = useState(false);
 
   const { config } = usePluginConfig();
-  const apiUrl = config?.api_url?.endsWith('/') ? `${config?.api_url}forms` : `${config?.api_url}/forms`;
+  const apiUrl = config?.api_url?.endsWith('/')
+    ? `${config?.api_url}public/forms/`
```

Comment:
Please remove the forward slash following the URL, as it is causing an issue.

```
SyntaxError {
    stack: 'SyntaxError: Unexpected token \'<\', "<!DOCTYPE "... is not valid JSON',
    message: 'Unexpected token \'<\', "<!DOCTYPE "... is not valid JSON'
  }
```

Reply:
Open Forms has a hard requirement for a trailing slash for this endpoint at the moment. There is a fix for this ready but I doubt this has been deployed on the Utrecht forms yet (https://github.com/open-formulieren/open-forms/commit/244ff0c99bb49c8a2ecd632cc65b52b93a9c9958)

Reply:
The issue has been resolved by the OpenForms API, but we need to confirm with Utrecht whether they have deployed the patched version.

https://github.com/open-formulieren/open-forms/commit/244ff0c99bb49c8a2ecd632cc65b52b93a9c9958

Reply:
This has been solved now since Open Forms 2.6.6